src/ui/cuicui/application-ui/battery/battery-indicator/battery-indicator.tsx

```diff
@@ -60,10 +62,10 @@ export const BatteryIndicator: React.FC<BatteryInfoProps> = ({
         className,
       )}
     >
-      {getBatteryIcon(batteryLevel, isCharging)}
+      {getBatteryIcon(level, isCharging)}
       <div className="flex flex-col">
-        <span className={`text-sm font-medium ${getTextColor(batteryLevel)}`}>
-          {batteryLevel}%
+        <span className={`text-sm font-medium ${getTextColor(level)}`}>
+          {level !== null ? `${batteryLevel}%` : 'Unavailable'}
```

Comment:
The expression `{level !== null ? "" : ""}` can be simplified to `{level ? "" : ""}` to eliminate unnecessary complexity.

Reply:
Well spotted